www/branch.js

```diff
@@ -263,65 +270,51 @@ Branch.prototype.createBranchUniversalObject = function (options) {
              *    |  $blackberry_url   |   String   |   Blackberry URL  |
              *    | $windows_phone_url |   String   |  Kindle Fire URL  |
              *    -------------------------------------------------------
-             */
```

Comment:
I think the old style here was fine, and likely more flexible since you can define the listeners flexibly instead of in one big callback object. It should be fine for a callback to be null as well as we should handle null checks in the Java code (see comment above).